yarn/pom.xml

```diff
@@ -109,6 +109,16 @@
       <properties>
         <jersey.version>1.9</jersey.version>
       </properties>
+      <!-- Enable hadoop-2.2 profile when the Hadoop version is 2.2.0, which is now 
+      the default. Those building against 2.2.x or some distro-specific variation of 
+      that string would still have to manually enable -Phadoop-2.2.
+      -->
+      <activation>
+       <property>
+        <name>hadoop.version</name>
```

Comment:
> Can we just fold this config into the main POM config for yarn?

I guess that could work. If there's any discrepancy in the versions in other hadoop profiles, we can override the versions later, but adding the dependencies for non-hadoop-2.2 is at worst redundant.